kaniko/kaniko.yaml

```diff
@@ -30,4 +32,5 @@ spec:
     command:
     - /kaniko/executor
     - --dockerfile=${inputs.params.DOCKERFILE}
+    - --context=/workspace/source/${inputs.params.CONTEXT} # The user does not need to care the workspace and the source.
```

Comment:
nice!! :D p.s. with [path templating](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#templating) (i.e. in this case `${inputs.resouces.source.path}/${inputs.params.CONTEXT}`) the author of the Task wouldnt have to worry about it either :D